content/10.diagnostics.md

```diff
@@ -511,6 +511,17 @@ Immunoassays, including serological tests, have much lower limits of detection t
 LOD between tests: 10.1056/NEJMcp2117115
 !--Add text citing papers from Ronan & Nils--> 
 
+### What lies ahead
+Diagnostic tools have played an important role during the COVID-19 pandemic. 
+Specifically, the results of SARS-CoV-2 diagnostic tests (typically qPCR or LFT-based tests) have been used to estimate the number of infections in the general population, thus informing public health strategies around the globe [@doi:10.1038/s41576-021-00360-w]. 
+During the surges caused by the different SARS-CoV-2 variants between 2020 and 2021, government-sponsored efforts to conduct mass testing and to provide free diagnostic tests to the population were a common occurrence in many parts of the world [@doi:10.1136/bmj.n902; @doi:10.1177/0020852321997552; @doi:10.1016/j.worlddev.2020.105208]. 
+However, recent reports indicate that such public health policies are starting to change during 2022. 
+For example, it is known that the UK plans to dismantle its COVID-19 testing program and scale back its daily reporting requirements [@url:https://www.independent.co.uk/news/health/who-covid-testing-anil-soni-b2032884.html; @doi:10.1136/bmj.o562]. 
+A similar approach can be seen in the US as well, where multiple state-run testing facilities are closing, despite some groups advocating to keep them open [@url:https://www.9news.com/article/news/health/coronavirus/local-groups-continue-push-for-covid-testing-and-vaccinations/73-bbcd8384-d96a-425e-aaeb-16ac9f36e581; @url:https://www.sltrib.com/news/2022/02/18/utah-will-stop-daily].
+These ongoing changes in testing policy are likely to have a direct effect on how the pandemic is managed moving forward, because SARS-CoV-2 diagnostic tests can be used effectively to slow the spread of the disease only when 1) they are used to share testing results in a timely manner so that they can reasonably be used to approximate the number of infections in the population, and 2) those tests are easily accessible by the general public. 
+In this regard, it is important to mention that the multiple SARS-CoV-2 diagnostic tools we have presented through this paper lack any real benefit if they are difficult to obtain, or if their use on a limited basis results in biased data that would lead to ill-informed public health strategies. 
```

Comment:
```suggestion
In this regard, it is important to mention that the multiple SARS-CoV-2 diagnostic tools we have presented in this paper lack any real benefit if they are difficult to obtain, or if their limited use results in biased data that would lead to ill-informed public health strategies. 
```

Reply:
I'm not sure it is totally necessary, so feel free to ignore

Reply:
I like the changes

Reply:
@rando2 I think I have the outline of what I want to add regarding @RLordan's comment about schools. Do I start a PR to do this?  (asking because I'm getting lost on the number of PRs and issues and I see that the changes made within the last day are still not in the master branch). Also, thank you both for the congratulations!